moped-editor/src/auth/user.js

```diff
@@ -260,7 +252,8 @@ export const UserProvider = ({ children }) => {
             // We must populate userDatabaseData if it's null otherwise the query will fail
             // when users are redirected to their last route after a forced logout
             // (see MainLayout.js and DashboardLayout.js for previous route restoration handling).
-            await initializeUserDBObject(session);
```

Comment:
This function used to fetch the user row and set it. It now only fetches it, and I forgot to make the update to set it in this part of the code.